src/plugins/notebook/utils/notebook-storage.js

```diff
@@ -54,6 +54,12 @@ export function getDefaultNotebook() {
     return JSON.parse(notebookStorage);
 }
 
+export function getDefaultNotebookObject(openmct) {
+    const defaultNotebook = getDefaultNotebook();
+
+    return defaultNotebook && openmct.objects.get(defaultNotebook.identifier);
```

Comment:
We might be able to avoid some subsequent boilerplate code if we always return a `Promise`, rather than a promise or `undefined`.

eg.
```JavaScript
if (defaultNotebook === undefined) {
    return Promise.resolve();
} else {
    return openmct.objects.get(defaultNotebook.identifier);
}
```